docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -328,7 +460,50 @@ Or to set the column ``my_new_partition_column`` as a partition column on a tabl
 
 The current values of a table's properties can be shown using :doc:`SHOW CREATE TABLE </sql/show-create-table>`.
 
-.. _iceberg-type-mapping:
+COMMENT
+~~~~~~~
+
+The Iceberg connector supports setting comments on the following objects:
+
+- tables
+- views
+- table columns
+
+The connector uses the command :doc:`COMMENT </sql/comment>` for setting
+comments.
+
+Setting comments on tables or columns is not supported in case that the
+Iceberg connector is using the :ref:`Glue catalog <iceberg-glue-catalog>`.
+
+.. _iceberg-data-management:
+
+Data management
+^^^^^^^^^^^^^^^
+
+The :ref:`sql-data-management` functionality includes support for ``INSERT``,
```

Comment:
Thats all there is .. so do we want this listed here again?

Reply:
Mind the fact that `* :doc:`/sql/truncate`` is missing from the listing.